package.json

```diff
@@ -21,13 +21,14 @@
     "homepage": "https://github.com/Exabyte-io/materials-designer",
```

Comment:
updated to allow the materials designer to run locally. Changes from https://github.com/Exabyte-io/materials-designer/pull/25/files